tests/snapshots/execution_spec__test-null-in-object.md_test_0.snap

```diff
@@ -0,0 +1,18 @@
+---
+source: tests/execution_spec.rs
+expression: response
+---
+{
+  "status": 200,
+  "headers": {
+    "content-type": "application/json"
+  },
+  "body": {
+    "data": {
+      "hi": {
```

Comment:
this is not quite right because the `hi` value should be null itself.

Imagine if nested fields were marked as required in this case we'll get an error although this is valid to have hi as null

Reply:
Let me check the graphql spec.